cypress.json

```diff
@@ -16,6 +16,11 @@
     "password": "admin",
     "ENDPOINT_WITH_PROXY": false,
     "MANAGED_SERVICE_ENDPOINT": false,
-    "VISBUILDER_ENABLED": false
+    "VISBUILDER_ENABLED": false,
+    "DATASOURCE_MANAGEMENT_ENABLED": false,
+    "OSD_TEST_DOMAIN_ENDPOINT_URL": "https://test",
```

Comment:
do we really need the endpoint as constant in cypress.json? I would prefer either use github secret if it's actual endpoint or constant string in test cases

Reply:
In my local, I used environment variable to define the URL and the current `OSD_TEST_DOMAIN_ENDPOINT_URL`  is a dummy url. We will create an official test domain later and use github secret then.

Reply:
Interesting problem! I am ok to move forward for now. 

We may need to brainstorm a bit how to support such when the tests run from our org's infra team https://github.com/opensearch-project/opensearch-build cc @seraphjiang @kavilla @CCongWang @kristenTian